apps/galaxy/src/modules/auth/SignInForm/SignInForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { ErrorMessage } from "@hookform/error-message";
+import useSignIn from "../../../services/signin/useSignIn";
```

Comment:
I think we have path mapping set up in this project. You can import this from `@src/services/signin/useSignin`.